### PR TITLE
Fix learning rate scheduler parameters

### PR DESCRIPTION
**Pull request by LoserCheems**
Adjust the learning rate scheduler configuration by replacing `decay_ratio` and `warmup_ratio` with `decay_steps` and `warmup_steps` for improved performance.

### CODE REVIEW

File: examples/pretraining/configs/Doge-20M.yaml [hunk 1]
```diff
@@ -55,9 +55,9 @@ lr_scheduler_type: warmup_stable_decay
 lr_scheduler_kwargs:
   warmup_type: linear
   decay_type: linear
-  decay_ratio: 0.1
+  decay_steps: 800
   min_lr_ratio: 0.0
-warmup_ratio: 0.1
```
Comment (LoserCheems) [inline examples/pretraining/configs/Doge-20M.yaml:60]: `lr_scheduler_kwargs` can't passing decay ratio, so change them to `steps`.
